charts/spring-service/values.yaml

```diff
@@ -75,3 +75,7 @@ alertingRules: []
 #   for: 5m # optional
 #   description: serviceName reports 95 percentile for Project Fields ACL endpoint was slower than 200ms in the last 1 hour # use actual serviceName for alert routing here
 #   playbook_url: http://xconfluence/x/aBcDeFg
+
+voluntaryDisruptions:
```

Comment:
Maybe call this `disruptionBudget` so that one can map it directly to the Kubernetes term without having to look into the chart what exactly `voluntaryDisruptions` does mean.